nix-workspace/flake.nix

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
**question:** are there already thoughts about how to keep this file up to date?